draft-ietf-lamps-dilithium-certificates.md

```diff
@@ -585,13 +585,16 @@ that do not perform this seed consistency check avoid keygen
 and compare operations, but are unable to ensure that the `seed` and
 `expandedKey` match.
 
-If the check is done and the seed and the `expandedKey` are not consistent,
+If the check is done and the `seed` and the `expandedKey` are not consistent,
 the recipient MUST reject the private key as malformed.
 
 The seed consistency check consists of regenerating the expanded form from
 the seed via `ML-DSA.KeyGen_internal` and ensuring it is bytewise equal to
 the value presented in the private key.
 
+{{example-bad}} includes some examples of inconsistent sees and expanded private
```

Comment:
```suggestion
{{example-bad}} includes some examples of inconsistent seeds and expanded private
```

Reply:
Perhaps here also worth noting that examples "2" and "3" don't include a seed, the inconsistency is entirely within the expanded key alone, by virtue of the public key, that can (and perhaps should on import) be computed from it, not matching corresponding data in the private key (the `tr` hash and/or the low bits vector `t_0`).